drivers/sensor/lis2dh/lis2dh.h

```diff
@@ -303,6 +305,21 @@ static inline int lis2dh_burst_write(struct device *dev, u8_t start_addr,
 
 	return lis2dh_spi_access(lis2dh, start_addr, buf, num_bytes);
 #elif defined(DT_ST_LIS2DH_BUS_I2C)
+	/* NRF TWIM is default and does not support burst write.  We
```

Comment:
Can we add a reference to issue #20154. in the comment block.

Reply:
Done